.github/release-drafter.yml

```diff
@@ -28,6 +28,10 @@ categories:
     collapse-after: 10
     labels:
       - bug
+  - title: 🗂️ Database Changes
+    collapse-after: 1
```

Comment:
Since the presence of at least 1 PR with a migration is all we really need to know, this should help minimize the real estate that this section takes up in the release notes.